plugins/catalog-backend-module-backstage-openapi/config.d.ts

```diff
@@ -29,6 +29,11 @@ export interface Config {
          * Properties to override on the final entity object.
          */
         entityOverrides?: object;
+        /**
+         * The format of the definition.
+         * @defaultValue json
+         */
+        definitionFormat: 'json' | 'yaml';
```

Comment:
Does this need to be `definitionFormat?: 'json' | 'yaml'` otherwise it might fail schema validation?

Reply:
Good catch, thanks!

Reply:
```suggestion
        definitionFormat?: 'json' | 'yaml';
```